[update] connect to Timescale Cloud (#1685)
timescale/docs: timescaledb/how-to-guides/connecting/about-psql.md

psql postgres://<USERNAME>@<HOSTNAME>:<PORT>/<DATABASENAME>?sslmode=require
```

> If you configured Timescale Cloud service to connect with stricter [SSL mode] [ssl-mode],

Reviewer: ```suggestion
If you configured your Timescale Cloud service to connect using [SSL mode] [ssl-mode],
```

> then use:

Reviewer: ```suggestion
use:
```

```shell
    psql "postgres://tsdbadmin@$SERVICE_URL_WITH_PORT/tsdb?sslmode=verify-full"
    

> ```

Reviewer: The spacing is out here, and it should be `bash` not `shell`, I think.

Author: done!!!
